src/psee_evaluator.py

```diff
@@ -20,6 +20,11 @@ def evaluate_folders(dt_folder, gt_folder):
     result_boxes_list = [reformat_boxes(p) for p in result_boxes_list]
     gt_boxes_list = [reformat_boxes(p) for p in gt_boxes_list]
 
+    min_box_diag = 60 if camera == 'GEN4' else 30
+    min_box_side = 20 if camera == 'GEN1' else 10
+
+    filter_fn = lambda x:filter_boxes(x, int(1e5), min_box_diag, min_box_side)
```

Comment:
this doesn't seem to be used,

maybe don't use the map ?

Reply:
ah thanks for that catch. the map works okay, if i specify the right function!